open_dread_rando/files/schema.json

```diff
@@ -313,9 +313,19 @@
                                     "type": "boolean",
                                     "default": false,
                                     "description": "Enables an in-game death counter shown in the player's HUD"
+                                },
+                                "room_ids": {
+                                    "type": "string",
+                                    "default": "NEVER",
```

Comment:
We've already have other enums are they're all lower case. Let's be consistent.

Also, use the `enum` key of the schema and list all possible values.